index.html

```diff
@@ -212,6 +212,58 @@ <h3>Multi-Post Strories</h3>
           </div>
         </div>
       </section>
+      <section class="contacts">
+        <h1>CONTACT US</h1>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form action="https://formspree.io/f/xzbodpbq" method="post">
+          <input
+            type="text"
+            name="name"
+            id="full-name"
+            placeholder="Text input"
+            required
+            maxlength="30"
+          />
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="jeremy@gmail.com"
+            required
+          />
+          <textarea
+            name="message"
+            id="message"
+            placeholder="Enter text here"
+          ></textarea>
```

Comment:
- [ ] GOOD PROJECT 🏆🔥.  Your project is nearly complete ⚖️. I am sure you forgot it accidentally because other things are working in form 🥲. The thing is max length and to fix it please add this code and done ✅🍀